Apps/Examples/Examples/All Examples/CustomSymbolAnnotationsExample.swift

```diff
@@ -0,0 +1,339 @@
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+        }
+
+        // Left-hand pin
+        if let image =  UIImage(named: "AnnotationLeftHanded") {
+            let regularAnnotationImage = image.tint(annotationColor)
+
+            style.setStyleImage(image: regularAnnotationImage,
+                                with: "AnnotationLeftHanded",
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+
+            let highlightedAnnotationImage = image.tint(annotationHighlightedColor)
+            style.setStyleImage(image: highlightedAnnotationImage,
+                                with: "AnnotationLeftHanded-Highlighted",
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+        }
+    }
+
+    static let annotations = "annotations"
+
+    private func addDebugFeatures() -> FeatureCollection {
+        var features = [Feature]()
+        let featureList = [
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.714203, -74.006314), highlighted: false, sortOrder: 0, tailPosition: .left, label: "Chambers & Broadway - Lefthand Stem", imageName: "AnnotationLeftHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.707918, -74.006008), highlighted: false, sortOrder: 0, tailPosition: .right, label: "Cliff & John - Righthand Stem", imageName: "AnnotationRightHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.716281, -74.004526), highlighted: true, sortOrder: 1, tailPosition: .right, label: "Broadway & Worth - Right Highlighted", imageName: "AnnotationRightHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.710194, -74.004248), highlighted: true, sortOrder: 1, tailPosition: .left, label: "Spruce & Gold - Left Highlighted", imageName: "AnnotationLeftHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.7128, -74.0060), highlighted: true, sortOrder: 2, tailPosition: .center, label: "City Hall - Centered Highlighted", imageName: "AnnotationCentered-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.711427, -74.008614), highlighted: false, sortOrder: 3, tailPosition: .center, label: "Broadway & Vesey - Centered Stem", imageName: "AnnotationCentered")
+        ]
+
+        for (index, feature) in featureList.enumerated() {
+            var featurePoint = Feature(Point(feature.coordinate))
+
+            // set the feature attributes which will be used in styling the symbol style layer
```

Comment:
nit: We should be consistent about comment structure. Could you update comments to use sentence casing with periods at the end?